### PR TITLE
Fixed Social media buttons on share susi menu

### LINKED ISSUE #3326: Fixing issues regarding social icons in 'Share about SUSI' menu
**Actual Behavior**
Here are few problems in this menu :
1. The text box does not cover problem space.
2. Hovering on social media icons works in Firefox but not on chrome.
3. The hover actions(in Firefox) and click actions doesn't match.

**Expected Behavior**

1. Text box must cover at least 90% of the box
2. Hover actions must be same for both the browsers, in chrome no hovering action is seen.
3. Same needs to be corrected

**Steps to reproduce it**

Goto : Settings->Share About SUSI->

1. Hover over the social media buttons and press left on them(keep it pressed for 3). SS have been added.

**Screenshots of the issue**

Hover + Click :
Firefox click and hover conflict

On hovering in firefox:
Firefox hover vs google chrome hover
No action on hovering in chrome:
Firefox hover vs google chrome hover

Textbox size :
Textbox wrap

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #3328: Non-uniformity in share susi menu
**Actual Behaviour**

Share Susi.Ai menu is not formatted as other menus are.

**Expected Behaviour**

Uniformity must be their in all menu present in settings.

**Steps to reproduce it**

Goto - Settings->share susi ai

**Screen Shots:** 
Screenshot from 2020-01-19 18-05-31
Screenshot from 2020-01-19 18-05-34

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by tobehonest**
Fixes #3328 and Fixes #3326

Changes: [Fixed uniformity in Susi share menu and others
Fixed Social icons in Susi share menu
Fixed hovering action of these in chrome
Fixed overlapping of on hover and click actions]

Screenshots of the change: 
Link : http://pr3342foss.surge.sh/settings

Susi Share menu and On Click :
Screenshot from 2020-01-19 18-12-45

On hover:
Screenshot from 2020-01-19 18-11-22

### CODE REVIEW

**Comment by tobehonest** [comment]
> Make a surge link to view your changes

http://pr3342foss.surge.sh/settings

**Comment by tobehonest** [comment]
@Dilshaad21 Please review the pr , I have added surge link.

**Comment by tobehonest** [comment]
@akshatnitd Please review this pr.

File: src/components/Settings/ShareOnSocialMedia.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import _IconButton from '@material-ui/core/IconButton';
-import DialogTitle from '@material-ui/core/DialogTitle';
+// import DialogTitle from '@material-ui/core/DialogTitle';
```
Comment (akshatnitd) [inline src/components/Settings/ShareOnSocialMedia.js]: Remove comment

File: src/components/Settings/ShareOnSocialMedia.js [hunk 2]
```diff
@@ -74,65 +81,80 @@ class ShareOnSocialMedia extends Component {
...
-              url={this.state.shareUrl}
-              quote={this.state.title}
+        <SettingsTabWrapper heading="Share about SUSI.AI">
+          <CopyWithButton
+            value={this.state.shareText}
+            width={this.props.width}
+            handleChange={this.handleChange}
+          />
+          <ShareIconContainer>
+            <IconButton
```
Comment (akshatnitd) [inline src/components/Settings/ShareOnSocialMedia.js]: Make a common IconButton and reuse it, instead of repeating the code.

Comment (tobehonest) [reply src/components/Settings/ShareOnSocialMedia.js]: I have made these changes. Please re-review.

**Comment by tobehonest** [comment]
@akshatnitd I have made the changes as suggested by you.

**Comment by tobehonest** [comment]
> squash your commits.

@samagragupta I have done that , please review.

**Pull request merged**